app/decorators/article_decorator.rb

```diff
@@ -1,6 +1,11 @@
 class ArticleDecorator < ApplicationDecorator
   delegate_all
 
+  def long_published_at
+    relevant_date = crossposted_at.present? ? crossposted_at : published_at
```

Comment:
This can be simplified
```suggestion
    relevant_date = crossposted_at.presence || published_at
```